### DISCUSSION TITLE
USB_VCP support for ESP32S3

### DISCUSSION THREAD

**Discussion by realvarx** [micropython/micropython#14250] [ESP32]
Is [USB_VCP](https://docs.micropython.org/en/latest/library/pyb.USB_VCP.html) supported on ESP32S3? Are there other alternatives to tx-rx data through the USB of the board (without using any additional pins)? 

I saw [this answer](https://github.com/micropython/micropython/issues/8582#issuecomment-1107455802) by @jimmo , I don't know if the situation might have changed, or if he was talking about something different.

Thanks in advance :slightly_smiling_face:

**Comment by ned-pcs** [comment]
The ESP32-S3 does in fact present its REPL via a USB-CDC (virtual COM port). So you're free to use it for transmitting and receiving data.

**Reply by realvarx** [reply]
Ohh I see. Thanks!
